sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -5,8 +5,6 @@ import ContrastIcon from "svg/icon-contrast.svg";
 import DownloadIcon from "svg/icon-download.svg";
 import PaletteIcon from "svg/icon-palette.svg";
 import PlayIcon from "svg/icon-play.svg";
-import ScreenIcon from "svg/icon-screen.svg";
```

Comment:
Should these files be removed as well? Doesn't look like they're being used anywhere else in the project.

Reply:
I removed the ones that are not being used